<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sunlab_gui</class>
 <widget class="QDialog" name="sunlab_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sunlab</string>
  </property>
  <widget class="QLineEdit" name="id_entry">
   <property name="geometry">
    <rect>
     <x>142</x>
     <y>120</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="swipe_in_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>swipe in</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
